### PR TITLE
22313-Packages-are-created-for-extension-methods-that-does-not-are-exactly-the-package-name

### PR DESCRIPTION
**Pull request by tesonep**
Fixing the lookup of extension packages that are not named exactly as the package.

### CODE REVIEW

File: src/Monticello-Tests/MCWorkingCopyForExtensions.class.st [hunk 1]
```diff
@@ -0,0 +1,43 @@
+Class {
+	#name : #MCWorkingCopyForExtensions,
```
Comment (dionisiydk) [inline src/Monticello-Tests/MCWorkingCopyForExtensions.class.st:2]: The Tests suffix is missing. It should be MCWorkingCopyForExtensionsTests

Comment (astares) [reply src/Monticello-Tests/MCWorkingCopyForExtensions.class.st:2]: and singular - MCWorkingCopyForExtensionsTest

**Comment by dionisiydk** [comment]
It would be nice to have one convention everywhere. But 50% of test cases
in system follow Tests suffix (and me too)

ср, 22 авг. 2018 г., 7:50 Astares <notifications@github.com>:

**Comment by astares** [comment]
- In the standard image 78% follow the singular convention, only 
        22% end with plural "Tests"
     - If you would follow this also in Calypso we would even have a 
       ratio of 89% compared to 11%

**Comment by astares** [comment]
Any new on this one?

**Comment by demarey** [comment]
I did a new pull request with the name changes #1741.
I'm now closing this one

**Pull request closed without merging**